RNTester/RNTester/AppDelegate.mm

```diff
@@ -89,8 +89,11 @@ - (NSURL *)sourceURLForBridge:(__unused RCTBridge *)bridge
 {
   NSString *bundlePrefix = [[[NSBundle mainBundle] infoDictionary] valueForKey:@"RN_BUNDLE_PREFIX"];
   NSString *bundleRoot = [NSString stringWithFormat:@"%@RNTester/js/RNTesterApp.ios", bundlePrefix];
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:bundleRoot
-                                                        fallbackResource:nil];
+  NSURL *jsUrl = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:bundleRoot fallbackResource:nil];
+#if DEBUG
+  [[RCTBundleURLProvider sharedSettings] setJsLocation:jsUrl.host];
```

Comment:
More explanation may be useful - this doesn't seem like the code we're calling is debug only code so why does calling this in debug fix the issue and why don't we have to call this in ship?